helm/local.py

```diff
@@ -71,3 +73,6 @@
     },
     "dummy": {"BACKEND": "django.core.cache.backends.dummy.DummyCache"},
 }
+
+IAB_FEED_URL = "https://www.ietf.org/blog/iab/feed"
+IAB_IETF_BLOG_URL = "https://www.ietf.org/blog/iab/"
```

Comment:
My understanding is that they are already in `ietf/settings/base.py`?

Reply:
Good catch.
I'll remove that.